test/show.jl

```diff
@@ -5,8 +5,8 @@ using LinearAlgebra, SparseArrays
 # For curmod_*
 include("testenv.jl")
 
-replstr(x) = sprint((io,x) -> show(IOContext(io, :limit => true, :displaysize => (24, 80)), MIME("text/plain"), x), x)
-showstr(x) = sprint((io,x) -> show(IOContext(io, :limit => true, :displaysize => (24, 80)), x), x)
+replstr(x, KV::Pair...) = sprint((io,x) -> show(IOContext(io, :limit => true, :displaysize => (24, 80), KV...), MIME("text/plain"), x), x)
```

Comment:
Minor point: upper case `KV` generally refers to the type, not the value.

Reply:
Ah, yes you are right, I was merely copying from `IOContext` methods in base/show.jl, will update.